description_identifier_types.yml

```diff
@@ -0,0 +1,10 @@
+# Description identifier type validation
+
+identifier: # expand list based on mapping
```

Comment:
Can we create a DescriptiveIdentifier class that is like a DescriptiveValue, but only supports these values as an enum?  Then we can just use the openapi validation and we don't have to write a custom validator.

https://github.com/sul-dlss/cocina-models/blob/main/openapi.yml#L1685-L1689